index.js

```diff
@@ -77,7 +77,7 @@ module.exports = (input, opts) => {
 			{
 				title: 'Installing dependencies using npm',
 				enabled: () => opts.yarn === false,
-				task: () => exec('npm', ['install'])
+				task: () => exec('npm', ['install', '--no-package-lock'])
```

Comment:
Not sure if we should handle outdated lockfiles like we do with Yarn? Not sure how though, currently I just want to be able to use `np` again on Node 8 without the lockfile getting in my way.